modules/filtlong/main.nf

```diff
@@ -27,7 +27,8 @@ process FILTLONG {
 
     script:
     def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
-    def short_reads = meta.single_end ? "-1 $shortreads" : "-1 ${shortreads[0]} -2 ${shortreads[1]}"
+    def short_reads = '' // Start with empty value in case there are no short reads.
+    if (shortreads) short_reads = meta.single_end ? "-1 $shortreads" : "-1 ${shortreads[0]} -2 ${shortreads[1]}"
```

Comment:
Thanks for finding my mistake! I'd rather propose
```suggestion
    def short_reads = shortreads ? ( meta.single_end ? "-1 $shortreads" : "-1 ${shortreads[0]} -2 ${shortreads[1]}" ) : ""
```
following the pattern in the [spades module](https://github.com/nf-core/modules/blob/45985ff6f01c6e2e229b665ba45b159bd925513d/modules/spades/main.nf#L37).